src/richie/apps/courses/templates/courses/cms/category_detail.html

```diff
@@ -1,6 +1,12 @@
 {% extends "richie/fullwidth.html" %}
 {% load cms_tags extra_tags i18n pagination_tags thumbnail %}
 
+{% block meta_html_default %}
+    {% if not current_page|is_empty_placeholder:"description" %}
+        <meta name="description" content="{% show_placeholder 'description' current_page %}" />
```

Comment:
There might be a problem with this field as its content is filled from CKEditor :disappointed_relieved: It will be html and may also be a bit long. Should we clean it and trunk it with a template filter?
Same for other object maybe...

Reply:
or a template tag:
```django
{% meta_format 200 %}{% show_placeholder 'description' current_page %}{% endmeta_format %}
```

Because I don't know how to apply a template filter to the "show_placeholder" tag as it does not have an option to assign its output to a variable?

Reply:
Now, I've included the `striptags` django filter that filters the html tags.